src/messaging/serializers.py

```diff
@@ -16,14 +16,21 @@ class Meta:
 
 
 class ThreadSerializer(WritableNestedModelSerializer):
-	profile_1 = ProfileSerializer()
-	profile_2 = ProfileSerializer()
 	recent_message = serializers.SerializerMethodField()
-
+	other_profile = serializers.SerializerMethodField()
+	
 	def get_recent_message(self, obj):
 		return MessageSerializer(Message.objects.filter(thread__id=obj.id).order_by('timestamp').last()).data
 
+	def get_other_profile(self, obj):
+		my_user = self.context['request'].user
```

Comment:
I've personally never used this before. can you have a sane default in case the request key isn't in context?